.env.local.example

```diff
@@ -1,19 +1,33 @@
 # RPC configuration
-# Follow the format: RPC_URL_{chainId}={rpcUrl}
-
-RPC_URL_1=https://ethereum-rpc.publicnode.com
-RPC_URL_8453=https://base-rpc.publicnode.com
-RPC_URL_59144=https://linea-rpc.publicnode.com
-
-# Identify which indexer plugin to activate (see `src/plugins` for available plugins)
-
-ACTIVE_PLUGIN=base.eth
+# For the RPC URL of each chain, follow the format: RPC_URL_{chainId}={rpcUrl}
+RPC_URL_1=https://eth.drpc.org
+RPC_URL_8453=https://base.drpc.org
+RPC_URL_59144=https://linea.drpc.org
```

Comment:
@shrugs @lightwalker-eth we now have the dRPC URLs as defaults for public RPC nodes